README.md

```diff
@@ -84,9 +84,10 @@ The current implementation creates a physical plan directly from the PromQL abst
 
 These are the latest benchmarks captured on an Apple M1 Pro processor.
 
-Note that memory usage is higher when executing a query with parallelism greater than 1. This is due to the fact that the engine is able to execute multiple operations at once (e.g. decode chunks from multiple series at the same time), which requires using independent buffers for each parallel operation. 
+Note that memory usage is higher when executing a query with parallelism greater than 1. This is due to the fact that the engine is able to execute multiple operations at once (e.g. decode chunks from multiple series at the same time), which requires using independent buffers for each parallel operation.
 
 Single core benchmarks
+
```

Comment:
I wonder why this wasn't caught by the CI lint job before.

Reply:
Ah, I think this is due to the `require_clean_work_tree` function not being defined in Makefile, so diff isn't ever checked. You can add it from here https://github.com/thanos-io/thanos/blob/main/Makefile#L89! 🙂

Reply:
Thanks, this solves the issue 👍